data/Wooller_2002/metadata.yml

```diff
@@ -0,0 +1,200 @@
+    119°22'E). This park is a large area (350 000 ha) of native vegetation, over half
+    of which consists of a marine plain and associated vegetation systems in which
+    we worked. The three banksias studied occur as widespread, intermingled and major
+    components of this vegetation, each representing about 6-30% of canopy cover (Chapman
+    and Newbey 1995). Banksia baxteri is a shrub to 4 m tall; B. nutans and B. baueri
+    are both bushes to 1-2 m tall and 1-2 m wide. ^p A constellation of three sites
+    5-10 km apart, whose fire histories were known from personal experience, were
+    monitored from 1983 to 2001. One area (Site A) burnt in 1980, had not been burnt
+    for at least 20 years before that. Another area (Site B) unburnt for at least
+    40 years at the start of the study remained unburnt. At the third site (Site C),
+    also long unburnt at the start of the study, part of the area (Site C1) was burnt
+    in a wildfire in December 1989, while the remainder (Site C2) remained unburnt
+    until a management fire in May 1998. At each site, the number of inflorescences
+    per plant was counted in most years on at least 40 plants of each species, selected
+    at random. Thus, it was possible to monitor the onset of flowering at Site A over
+    19 years and at Site C over 12 and 3 years after fire and to compare them with
+    nearby, similar vegetation long unburnt (Site B). This method helped to reduce
+    the problems associated with overlapping chronosequences in which differences
+    between sites may obscure temporal patterns (Whelan 1995). For each of the three
+    species, the numbers of infructescences per plant were counted at Site C1 10 years
+    after fire, at Site A 20 years after fire and at Site B more than 40 years after
+    fire. At burnt sites, seedling density was determined from fifty x 1 m 2 quadrats
+    for the first 3 years after fire, then twenty x 6.25 m 2 quadrats up to 10 years
+    after fire; thereafter, total counts of all individuals of each species were made
+    within 50 x 50 m grids established for animal trapping. Each of the three species
+    had readily distinguishable leaves, even at the seedling stage, facilitating density
+    estimates.
+  original_file: none, transcribed from manuscript
+  notes: none
+locations:
+  Fitzgerald River National Park:
+    latitude (deg): -34.2
+    longitude (deg): 119.37
+    description: sclerophyllous Mediterranean shrubland and heathland
+    soil type: deep sand
+contexts: .na
+traits:
+- var_in: resprouting_capacity
+  unit_in: .na
+  trait_name: resprouting_capacity
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: reproductive_maturity
+  unit_in: a
+  trait_name: reproductive_maturity
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
```

Comment:
`value_type: mean` and `basis_of_value: measurement`?

Reply:
done